### PR TITLE
Known issue csi

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR updates a know issue related to self-hosted Palette installs

## Changed Pages

💻 [Preview URL for Page](https://deploy-preview-2889--docs-spectrocloud.netlify.app/release-notes/known-issues/)

## Jira Tickets

🎫 [DOC-1192](https://spectrocloud.atlassian.net/browse/DOC-1192)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._
- [ ] No. _Please leave a short comment below about why this PR cannot be backported._

[DOC-1192]: https://spectrocloud.atlassian.net/browse/DOC-1192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/release-notes/known-issues.md [hunk 1]
```diff
@@ -14,18 +14,19 @@ to review and stay informed about the status of known issues in Palette. As issu
...
+| Deploying a self-hosted Palette or VerteX, to a vSphere environment fails if vCenter has standalone hosts directly under a Datacenter. Persistent Volume (PV) provisioning fails due to an upstream issue with the vSphere Container Storage Interface (CSI) for all versions before v3.2.0. Palette and VerteX use the vSphere CSI version 3.12 internally | If you encounter the following error message: `'ProvisioningFailed failed to provision volume with StorageClass "spectro-storage-class". Error: failed to fetch hosts from entity ComputeResource:domain-xyz` then use the following workaround. Remove standalone hosts directly under the Datacenter from vCenter and allow the volume provisioning to complete. After the volume is provisioned, you can add the standalone hosts back. You can also use a service account that does not have access to the standalone hosts as the user that deployed Palette. | May 21, 2024   | Self-Hosted       |
```
[the remark this answers is not in the document]
Comment (brathina-spectro) [reply docs/docs-content/release-notes/known-issues.md:19]: Palette and VerteX use the vSphere CSI version 3.1.2 internally. A dot is missing between minor and patch version

Comment (brathina-spectro) [inline docs/docs-content/release-notes/known-issues.md:19]: > You can also use a service account that does not have access to the standalone hosts as the user that deployed Palette. 

The 2nd part is not clear, specifically - that deployed Palette. Should it be "is deploying self-hosted Palette or Vertex" to cover the users who are reading through the doc before attempting an install?

Comment (brathina-spectro) [inline docs/docs-content/release-notes/known-issues.md:19]: The other general comment is, this is also possible for other use case where a sphere cluster is deployed with the affected vsphere-csi storage driver. VOlume provisioning of the custom workloads will also experience the same issue.

File: docs/docs-content/release-notes/known-issues.md [hunk 2]
```diff
@@ -14,18 +14,19 @@ to review and stay informed about the status of known issues in Palette. As issu
...
+| During the cluster provisioning process of new edge clusters, the palette webhook pods may not always deploy successfully, causing the cluster to be stuck in the provisioning phase. This issue does not impact deployed clusters.                                                                                                                                                                                                                                                      | Review the [Palette Webhook Pods Fail to Start](../troubleshooting/edge.md#scenario---palette-webhook-pods-fail-to-start) troubleshooting guide for resolution steps.                                                                                                                                                                                                                                                                                                                                                                                                                                           | April 14, 2024 | Edge              |
```
Comment (lennessyy) [inline docs/docs-content/release-notes/known-issues.md:29]:
```suggestion
| During the cluster provisioning process of new edge clusters, the Palette webhook pods may not always deploy successfully, causing the cluster to be stuck in the provisioning phase. This issue does not impact deployed clusters.                                                                                                                                                                                                                                                      | Review the [Palette Webhook Pods Fail to Start](../troubleshooting/edge.md#scenario---palette-webhook-pods-fail-to-start) troubleshooting guide for resolution steps.                                                                                                                                                                                                                                                                                                                                                                                                                                           | April 14, 2024 | Edge              |
```

**Review by lennessyy — APPROVED** [review]

**Pull request merged**